.github/workflows/spelling.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Run spelling check
         run: |
           ./eng/common/scripts/check-spelling-in-changed-files.ps1 `
-              -CSpellConfigPath 'cSpell.json' `
+              -CSpellConfigPath 'cspell.json' `
```

Comment:
Can this not take in cspell.yml? Also do we even need to pass it in explicitly any longer won't it be discovered based on the file being checked?

Reply:
The current implementation of `check-spelling-for-changed-files.ps1` needs a cspell config JSON file (YAML not supported) to place a list of files to be scanned. Future versions will not require that as the `files` property has been removed in the latest version of cspell.